src/dataflow-types/src/client/controller/storage.rs

```diff
@@ -95,6 +95,14 @@ impl<'a, T: Timestamp + Lattice> StorageControllerMut<'a, T> {
         }
     }
 
+    pub async fn create_instance(&mut self) {
```

Comment:
Some sort of doc comment please! We don't have such a function in `controller/compute.rs`, so it isn't as clear what it does.

Reply:
I removed it and absorbed the command into creating the storage controller.